spec/forms/advanced_search_form_spec.rb

```diff
@@ -1,52 +1,66 @@
-# frozen_string_literal: true
-
 require 'rails_helper'
-
-RSpec.describe AdvancedSearchForm do
+RSpec.describe DeveloperSearchForm do
   describe '#search' do
-    context 'with no params' do
-      it 'returns no posts' do
-        create_list(:post, 2)
+    before do
```

Comment:
Can you read the following message?
https://reapra-tech.slack.com/archives/C9NTX8Z0A/p1522755906000146